bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/WJetsToLNu/W3JetsToLNu_13TeV-madgraphMLM-pythia8/W3JetsToLNu_13TeV-madgraphMLM-pythia8_proc_card.dat

```diff
@@ -0,0 +1,9 @@
+import model sm-ckm_no_b_mass
+
+generate p p > w-, w- > l- vl~ /h @0
+add process p p > w- j, w- > l- vl~ @1
+add process p p > w- j j, w- > l- vl~ @2
+add process p p > w- j j j, w- > l- vl~ @3
```

Comment:
as commented here
https://github.com/cms-sw/genproductions/pull/1474
are you sure you want to include 0j,1j,2j ?
why do you include only the W minus?!?

Reply:
Sorry for mistake, i talked to Anirban and then compare with previous cards https://github.com/cms-sw/genproductions/tree/pre2017/bin/MadGraph5_aMCatNLO/cards/production/13TeV/WJets_jetbinning_LO_MLM 
and i found that we need to include W+ and W- , we don't need to include 0j, 1j and 2j.
I will make some changes according to previous cards.